src/components/layout/AppHeader/AppHeader.tsx

```diff
@@ -1,10 +1,67 @@
 import './AppHeader.css'
+import '../../../assets/styles/flex-patterns.css'
+import { useState } from "react";
+import { useWindowSize } from "../../../states/windowSizeState.ts";
+import { AppNavbar } from "../AppNavbar/AppNavbar.tsx";
 
 export const AppHeader = () => {
+    const [isDropDownMenuVisible, setIsDropDownMenuVisible] = useState(false);
 
     return (
-        <header className="header">
-
+        <header className="header center-flex">
+            <div className="header__container space-between-row-center-flex">
+                <div
+                    className={isDropDownMenuVisible ?
+                        "header__logo-container space-between-row-center-flex menu-is-hovered" :
+                        "header__logo-container space-between-row-center-flex"
+                    }
+                    onMouseOver={() => {
+                        if (!useWindowSize.getState().isDesktopScreen) {
+                            setIsDropDownMenuVisible(true)
+                        }
+                    }}
+                    onMouseOut={() => {
+                        setIsDropDownMenuVisible(false)
+                    }}
```

Comment:
How do you handle case when user try open drop down menu from mobile device?

Reply:
Will be solved in the next contribution